tests/fixtures/noqa/noqa.py

```diff
@@ -693,6 +693,15 @@ def consecutive_yields():
 ]
 *numbers, = [4, 7]  # noqa: WPS356
 
+
+class AttributeGetter(object):  # noqa: WPS614
+
+    def __init__(self):
+        self.attribute = 1
+
+    def get_attribute(self):
+        return 1
```

Comment:
Can we replace this line with `anti_wps = ...`? Because `return` is not important here.

Reply:
What does `anti_wps` represent? Would the line become:
```
def get_attribute(self):
        anti_wps614 = 1
```
?